ui/components/app/menu-bar/account-options-menu.js

```diff
@@ -51,10 +54,14 @@ export default function AccountOptionsMenu({ anchorElement, onClose }) {
     },
   });
 
-  const keyring = useSelector(getCurrentKeyring);
-  const chainId = useSelector(getCurrentChainId);
-  const rpcPrefs = useSelector(getRpcPrefsForCurrentProvider);
-  const selectedIdentity = useSelector(getSelectedIdentity);
+  const blockExplorerLinkClickedEvent = useNewMetricEvent({
+    category: 'Navigation',
+    event: 'Clicked Block Explorer Link',
+    properties: {
+      link_type: 'Account Tracker',
```

Comment:
We still would want the domain here, so that we could segment the charts out by domain, yeah?

Reply:
We add the network as a property higher up:
![Screen Shot 2021-05-14 at 6 05 27 PM](https://user-images.githubusercontent.com/34557516/118339618-f91b7e00-b4de-11eb-9ed6-b13516405e58.png)

Reply:
Though perhaps we want the block explorer url in addition? What do you think?

Reply:
Sorry I meant the domain of the block explorer -- if possible we should do domain and then the rest of the path as separate properties I think

Reply:
@brad-decker added `block_explorer_domain `!